workers/unity/Packages/com.improbable.gdk.transformsynchronization/TransformSynchronizationHelper.cs

```diff
@@ -1,9 +1,43 @@
+using Improbable.Gdk.Core;
+using Improbable.Transform;
 using Unity.Entities;
 
 namespace Improbable.Gdk.TransformSynchronization
 {
-    public static class TransformSynchronizationSystemHelper
+    public static class TransformSynchronizationHelper
     {
+        public static EntityBuilder AddTransformSynchronizationComponents(this EntityBuilder entityBuilder, string writeAccess,
```

Comment:
Instead of these overloads, can you just inline the default quaternion?

Reply:
I can not because the default Quaternion appears to be (0,0,0,0), not (1,0,0,0) - and can't do `new Quaternion(1f, 0f, 0f, 0f)` because default parameter needs to be a compile-time constant

Reply:
Use `Quaternion.identity` - also, the identity Quaternion is `0,0,0,1`...

Reply:
as Jamie said, our Quaternions are wxyz but Unity uses xyzw

though I could just make the function take in Unity Quaternions and abstract away our Quaternions in the AddComponent method

Reply:
The arguments to this should just use the default Unity types, Vector3 and Quaternion. We can convert those inside the feature modules using the extensions methods.

Reply:
The physics tick and ticks per second shouldn't be set as arguments either. They should be set to 0 and the 1/fixedDeltaTime.

Reply:
discussed offline:

Quaternion.identity is not a compile-time constant :/ - so we need separate methods